dashboard_documentation.md

```diff
@@ -95,10 +183,13 @@ To create an Observation or Forecast, an associated site must already exist (see
 ### Create New Forecast
 {: .anchor}
 
-1.  Navigate to the Site listing page using the **Sites** link in the left
-	sidebar. Select the site for which you are adding a Forecast.
+Forecasts can be created for either a Site or an Aggregate.
+
+1.  Navigate to the sites or aggregates page using the sidebar. From the
+    applicable listing page, select the site or aggregate for which you
+    are adding a Forecast.
```

Comment:
Navigate to the sites or aggregates page using the sidebar, then select the site or aggregate for which you are adding a Forecast.